### DISCUSSION TITLE
Docker install fail with oracle DB

### DISCUSSION THREAD

**Discussion by sendaops** [nextgenhealthcare/connect#5482] [General]
Hi!
I'm using the Dockerfile (https://github.com/nextgenhealthcare/connect-docker) to install the app without modifications, just typing:

 docker run --env-file=envmirth.txt -p 8080:8080 nextgenhealthcare/connect:4.0-jdk

where "envmirth.txt" has my Oracle DB connection string.
I´m getting the error:
_ERROR 2022-10-19 14:13:09,546 [Main Server Thread] com.mirth.connect.server.Mirth: Failed to migrate database schema
com.mirth.connect.model.util.MigrationException: Failed to execute script: /oracle/oracle-database.sql_
I've attached the full log, I've checked permissions and everything is ok. Also I try to execute this script https://github.com/nextgenhealthcare/connect/blob/development/server/dbconf/oracle/oracle-database.sql but is not working either.
I've checked the container and there isn't any other script to try.

Any help will be appreciated.
[mirth2.log](https://github.com/nextgenhealthcare/connect/files/9821724/mirth2.log)

**Comment by jonbartels** [comment]
The log tells you the problem: `java.sql.SQLSyntaxErrorException: ORA-01031: insufficient privileges`

Fix the privileges for the Oracle user that Mirth is configured to use.

@roggss recently had similar Oracle questions at https://github.com/nextgenhealthcare/connect/discussions/5427 .

**Comment by sendaops** [comment]
Hi Jon, thanks but that issue wasn't resolve yet.

**Comment by roggss** [comment]
Hi, I even set the user to DBA role.. Here what is granted, i knew its too
much privs,, but I ran out of ideas.


grant connect to MIRTH3;
grant dba to MIRTH3;
grant resource to MIRTH3;
-- Grant/Revoke system privileges
grant alter any table to MIRTH3;
grant create any table to MIRTH3;
grant create any view to MIRTH3;
grant debug connect session to MIRTH3;
grant select any table to MIRTH3;
grant unlimited tablespace to MIRTH3;

**Reply by sendaops** [reply]
Thanks, I'll check that.

**Reply by sendaops** [reply]
Hi, it works perfect, thank you!

**Comment by roggss** [comment]
@sendaops can you share the solution please?

**Reply by roggss** [reply]
Ignore,, I thought you have other solution than mine.

anyway.. thanks for giving me this SQL link, i am asking this on my thread and no one cares :),, I will create these objects myself.

Do you know if there is a SQL file for version 3.9.. since yours is for version 4.1.

Thanks in advance

**Reply by roggss** [reply]
Ignore, found it :)
